dimod/discrete/discrete_quadratic_model.py

```diff
@@ -793,13 +813,20 @@ def to_numpy_vectors(self):
             :meth:`~DiscreteQuadraticModel.from_numpy_vectors`
 
         """
-        case_starts, linear_biases, quadratic = self._cydqm.to_numpy_vectors()
+        if not return_offset:
+            case_starts, linear_biases, quadratic = self._cydqm.to_numpy_vectors()
+
+            return DQMVectors(case_starts,
```

Comment:
so I think this is actually still a backwards compatibility break, since
```
starts, ldata, quadratic = dqm.to_numpy_vectors()
```
will break (I think).

Unfortunately I think we need to have two different `DQMVectors`, maybe one called `LegacyDQMVectors` or something

Reply:
I actually had both `DQMVectors` and `DQMVectorsWithOffset` at one point. But the above naming scheme makes sense along with the deprecation warning. Ended up changing it previously due to the default value feature of named tuples.